03/db.py

```diff
@@ -0,0 +1,76 @@
+from model import Task
+import sqlite3
+
+MY_TODO_TABLE = 'TODO_LIST'
+
+
+def create_db():
+    con = sqlite3.connect('./todos.db')
+    return con
+
+
+def init_db(con: sqlite3.Connection):
+    cursor = con.cursor()
+    cursor.execute(f'''
+        CREATE TABLE IF NOT EXISTS {MY_TODO_TABLE} (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            title TEXT,
+            content TEXT,
+            done BOOLEAN
+        )''')
+    cursor.close()
+
+
+
+def create_task(con: sqlite3.Connection, title: str, content: str):
+    cursor = con.cursor()
+    cursor.execute(f'''
+        INSERT INTO {MY_TODO_TABLE} (title, content, done)
+        VALUES ('{title}', '{content}', {False})
+    ''')
+    cursor.close()
+    con.commit()
+
+
+def read_task(con: sqlite3.Connection):
+    cursor = con.cursor()
+    cursor.execute(f'''
+        SELECT * FROM {MY_TODO_TABLE}
+    ''')
+    todos = cursor.fetchall()
+    cursor.close()
+    tasks = [Task(todo) for todo in todos]
+    return tasks
+
+
+def update_task(con: sqlite3.Connection, _id: int):
```

Comment:
```suggestion
def update_task(con: sqlite3.Connection, id_: int):
```

`_name` 형태는 파이썬에서 private member를 의미하기 때문에(접근제한자 개념이 없어서... 이런 컨벤션이) built-in name shadowing을 회피하기 위해서는 뒤쪽에(trailing) 언더스코어를 붙이는 편입니다. `id_`, `type_`, `dir_` 등